1_technical_problem_solving_and_analysis/problems/2_pangram.rb

```diff
@@ -0,0 +1,108 @@
+=begin
+A pangram is a phrase which contains every letter at least once,
+such as “the quick brown fox jumps over the lazy dog”.
+Write a function which determines if a given string is a pangram.
+Please find at least three different strategies for solving this problem
+=end
+
+# Pangram is a sentence that contains all the letters of the alphabet.
+# Method accepts string.
+# Method outputs boolean.
+# Get all the letters of the alphabet. Can either put in array or hash. Hash is faster
+class Pangram
+  attr_reader :alphabet_hash
+  attr_reader :alphabet_array
+  attr_reader :string_to_evaluate
+
+  def initialize(string_to_evaluate)
+    @string_to_evaluate = string_to_evaluate
+    @alphabet_hash = {
```

Comment:
Is there a way to initialize this hash with a bit less typing?  For example, would something like this work?

```
    @alphabet_hash = {}
    ('a'..'z').each do |letter|
        @alphabet_hash[letter] = false
    end
```